components/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/RequestExecutor.java

```diff
@@ -108,6 +150,97 @@ private void validateExecutionParams() throws InternalWorkflowException {
+            if (PROCESS_UUID.equals(workflowDetail.getLocalName())) {
+                workFlowRequest.setRequestId(workflowDetail.getText());
+            }
+            if(TASK_INITIATOR.equals(workflowDetail.getLocalName())){
+                Variable variable = new Variable(TASK_INITIATOR, workflowDetail.getText());
+                parameterArray.add(variable);
+            }
+            if (WORKFLOW_PARAMETERS.equals(workflowDetail.getLocalName())) {
+                String parameterName = null;
+                Iterator parameters = workflowDetail.getChildElements();
+
+                while (parameters.hasNext()) {
+                    OMElementImpl parameter = (OMElementImpl) parameters.next();
+                    Iterator parameterValues = parameter.getChildElements();
+                    Iterator parameterAttributes = parameter.getAllAttributes();
+
+                    while (parameters.hasNext()) {
+                        OMAttributeImpl parameterAttribute = (OMAttributeImpl) parameterAttributes.next();
+                        if (NAME_PARAMETER_ATTRIBUTE.equals(parameterAttribute.getLocalName())) {
+                            parameterName = parameterAttribute.getAttributeValue();
+                            break;
+                        }
+                    }
+
+                    while (parameterValues.hasNext()) {
+                        OMElementImpl parameterValue = (OMElementImpl) parameterValues.next();
+                        Iterator parameterItemValues = parameterValue.getChildElements();
+                        while (parameterItemValues.hasNext()) {
+                            OMElementImpl parameterItemValue = (OMElementImpl) parameterItemValues.next();
+                            if(!REQUEST_ID.equals(parameterName)){
+                                Variable variable = new Variable(parameterName, parameterItemValue.getText());
+                                parameterArray.add(variable);
+                            }
+                        }
+                    }
+                }
+            }
+
+        }
+        workFlowRequest.setVariables(parameterArray);
+
+        return workFlowRequest;
+    }
+
+    private void callMediator(String workflowRequest) throws IOException {
+
+        String workflowMediatorURL = bpsProfile.getManagerHostURL();
+
+        URL url = new URL(workflowMediatorURL);
+        HttpURLConnection con = (HttpURLConnection) url.openConnection();
+
+        con.setRequestMethod(POST);
+        con.setRequestProperty(CONTENT_TYPE, APPLICATION_JSON);
+        con.setDoOutput(true);
+
+        try (OutputStream os = con.getOutputStream()) {
+            byte[] input = workflowRequest.getBytes(StandardCharsets.UTF_8);
+            os.write(input, 0, input.length);
+
+        }
+        int responseCode = con.getResponseCode();
+        System.out.println("POST Response Code :: " + responseCode);
```

Comment:
We can remove both lines.

Reply:
addressed the comment ref:https://github.com/wso2-extensions/identity-workflow-impl-bps/pull/70/commits/641aaf120ea3a609b753a77ae028dae66a9ffa98